cruise-control/src/test/java/com/linkedin/kafka/cruisecontrol/monitor/sampling/aggregator/KafkaPartitionMetricSampleAggregatorTest.java

```diff
@@ -81,11 +81,12 @@ public void testAggregate() throws NotEnoughValidWindowsException {
         Collection<Short> metricIds = KafkaMetricDef.resourceToMetricIds(resource);
         double expectedValue = (resource == Resource.DISK ?
             (NUM_WINDOWS - 1 - i) * 10 + MIN_SAMPLES_PER_WINDOW - 1 :
-            (NUM_WINDOWS - 1 - i) * 10 + (MIN_SAMPLES_PER_WINDOW - 1) / 2.0) * metricIds.size();
+            (NUM_WINDOWS - 1 - i) * 10 + (MIN_SAMPLES_PER_WINDOW - 1) / 2.0)
+            * (resource == Resource.CPU ? 0.01 : 1.0) * metricIds.size();
```

Comment:
Nit: Moving these hard coded values to static variables would make the code more clear.